.github/mergify.yml

```diff
@@ -0,0 +1,12 @@
+queue_rules:
+  - name: default
+    batch_size: 8
```

Comment:
Lets set the batch time slightly longer. With bors it was always a bit of a rush to get a batch going. Maybe `batch_max_wait_time: 60 s`?

Reply:
`bors` was probably less than 3 secs 😆 so I thought 30s would be more than enough for you! It's a trade off vs time to merge, but 30s extra wait time is probably not noticeable. I'll change it to `60s`.

Reply:
Updated in https://github.com/sigp/lighthouse/pull/4917/commits/2c476b38d2a72cb07284b30b549808dcf963b6a8